askbot/templates/editors/rich_text.html

```diff
@@ -30,7 +30,7 @@
                 django.jQuery = $;
             </script>
             <script type="text/javascript">
-                askbot['settings']['tinyMCEPlugins'] = JSON.parse('{{ settings.TINYMCE_PLUGINS|as_json }}');
+                askbot['settings']['tinyMCEPlugins'] = JSON.parse('{{ TINYMCE_PLUGINS|as_json }}');
```

Comment:
I've reproduced this issue. 'settings' is the context variable that is set by askbot/context.py. To solve the problem, I've passed it explicitly from upstream code using the rich_text.html template.

https://github.com/ASKBOT/askbot-devel/commit/928f75da62625b5d6a57c1a46f847a27bd879e53